### PR TITLE
use dependencies in Makefile

### PR DESCRIPTION
**Pull request by fgfm999**
use target dependency instead of calling another target explicitly,  avoid running another instance of make.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,33 +38,29 @@ clean-workspace:
...
 
 clean-build-cache:
 	rm -rf cache/build/*
 	echo "remove build cache."
 
 ### 2. main command
-init:
-	make create-folder
+
+init: create-folder prepare_files install build-image
```
Comment (RetricSu) [inline Makefile:53]:
```suggestion
init: create-folder prepare-files install build-image
```

File: Makefile [hunk 2]
```diff
@@ -38,33 +38,29 @@ clean-workspace:
...
+
+prepare_files:
```
Comment (RetricSu) [inline Makefile:55]:
```suggestion
prepare-files:
```

Comment (fgfm999) [reply Makefile]: done

File: Makefile (hunk 1 above)
Comment (fgfm999) [reply Makefile]: done

**Pull request merged**